### PR TITLE
gobrake-fasthttp middleware and example application

### PR DESCRIPTION
**Pull request by chimanjain**
Added gobrake-fasthttp middleware
Added sample application

### CODE REVIEW

**Review by andrew-werdna — APPROVED** [review]
Solid work, LGTM!

File: examples/fasthttp/README.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Beego integration
```
Comment (rbg) [inline examples/fasthttp/README.md:1]: ^---- typo ?

Comment (chimanjain) [reply examples/fasthttp/README.md:1]: Great catch

**Comment by chimanjain** [comment]
PTAL

**Pull request merged**